experiment.py

```diff
@@ -14,15 +14,37 @@
 DATA_DIR = './dataset'
 
 
+def read_settings(settings_path):
+    if os.path.exists(settings_path):
+        with open(settings_path, 'r') as f:
+            settings = json.load(f)
+    else:
+        settings = {
+            'normalize_continuous': True,
+            'one_hot_encode_features': True,
+            'impute_continuous': True,
+            # of shape [continious columns, lower bounds, upper bounds]
+            'bounds': None
+            # alternatively, pass in individual bounds
+            # lower_bound: [continuous cols]
+            # upper_bound: [continuous cols]
+        }
+    return settings
```

Comment:
Note: I added this new setting file so that the user can pass in CR/CL bounds as well as controlling normalization and one-hot encoding etc. (those are currently hard-coded)